services/brig/src/Brig/IO/Intra.hs

```diff
@@ -663,6 +665,16 @@ getTeamName tid = do
     req = paths ["i", "teams", toByteString' tid, "name"]
         . expect2xx
 
+getTeamLegalHoldStatus :: UserId -> TeamId -> AppIO ViewLegalHoldService
+getTeamLegalHoldStatus u tid = do
```

Comment:
I think this works, but you could also hit `get "/i/teams/:tid/features/legalhold"` and avoid the fake authentication.  No strong opinion from me here.

Reply:
Thanks!
Actually, this was a source of a bug: one galley integration test creates a user with no permissions, but here, I expect the call from brig to galley to succeed, which it doesn't when a user doesn't have permissions to view these settings. 
I changed this now to make use of the `/i/` endpoint.